### DISCUSSION TITLE
Default game volume at first-time setup is too high on desktop platforms

### DISCUSSION THREAD

**Discussion by Bruno5430** [ppy/osu#20387] [Q&A]
### Type

Cosmetic

### Bug description

When someone tries lazer for their first time, they get soundblasted by the osu! Triangles intro

I've seen a lot of people complaining about it from multiple places (YouTube, Discord, even the in-game chat), and experienced most of those reactions myself on discord voice calls (those goes like: *Triangles intro theme playing at the background* aaaaaaaah my ears!!!) (someone even uninstalled the game after getting jumpscared and didn't try it again to this day)

Most of them set the volume in a range of 50-20% (except for mobile players, as their speakers aren't loud enough)
I tried it on a clean install and i think 50% volume is a good default to stop people from getting soundblasted while keeping the volume high enough on most cases

### Version

2022.911.0-lazer

### Logs

[database.log](https://github.com/ppy/osu/files/9612028/database.log)
[network.log](https://github.com/ppy/osu/files/9612029/network.log)
[performance.log](https://github.com/ppy/osu/files/9612030/performance.log)
[runtime.log](https://github.com/ppy/osu/files/9612031/runtime.log)

**Comment by peppy** [comment]
Could you please provide screenshots of the audio mixers from each of these complaints?

We've just recently tested locally and at 100% it matches expectations of other apps on the system, so I'm very curious to see what settings the users getting "jumpscared" to the point of uninstallation have.

**Comment by Bruno5430** [comment]
I asked for screenshots and only got response from 1 guy at the moment
Tomorrow will probably have more
![unknown](https://user-images.githubusercontent.com/111712756/191392310-04a8563e-670f-4aef-8097-a78f604057d3.jpg)
![unknown2](https://user-images.githubusercontent.com/111712756/191392329-4177b41f-b981-4b94-941f-d7ce7ee3568a.png)
![unknown3](https://user-images.githubusercontent.com/111712756/191392343-f292e926-e7e3-48d6-a34e-db8e41f10216.png)

**Reply by Bruno5430** [reply]
note that every complaiant uses headphones and didn't had any issues with speakers

**Reply by peppy** [reply]
Minecraft should be equally loud, in such a setup. Does the user also set minecraft to low volume from within the game?

**Reply by Bruno5430** [reply]
The user declares that minecraft isn't as loud as lazer
![unknown4](https://user-images.githubusercontent.com/111712756/191394689-19095ca2-b482-4dee-927e-baa7b2f8a496.png)

**Reply by peppy** [reply]
yes well minecraft hardly has sound and the music tracks are super low volume at a track level, so that is t a great comparison. can you have them compare to a music player on their system at full volume?

**Comment by Bruno5430** [comment]
Here is mine
![osu_2022-09-20_23-01-44](https://user-images.githubusercontent.com/111712756/191397842-6158559e-a476-4baa-b30d-4b913b220220.jpg)
![2022-09-20_23-03](https://user-images.githubusercontent.com/111712756/191397994-1a3891bf-b7f3-43b2-b9da-87da63b9dd54.png)
![2022-09-20_23-03_1](https://user-images.githubusercontent.com/111712756/191398003-fe6d899d-49cb-4769-abd4-7b2327d0242d.png)

**Comment by Doksha** [comment]
It is basically necessary to set your volume to around 10-20%, the same issue is in stable. The first thing I tell people when they first install osu (stable or lazer) is to take off their headphones and scroll down as fast as possible on startup. I understand allowing higher volumes, but the default should be 25%
playing music on any other music player on full volume is not near as loud as osu on full volume.
The point is most people have this issue, whether or not it is *technically* the fault of the user, it needs to be fixed

**Reply by peppy** [reply]
Please tell us which app you're using to play music. We've tested all the standard apps (VLC, Apple Music etc.) and the volume matches. On every OS we've tested. Windows, macOS and linux.

**Comment by DigitalHypno** [comment]
During the usability testing that was run a few weeks ago, 3 of the players were installing lazer for the very first time. Of which, all 3 of those players encountered this same problem and had to lower their volume significantly.

This video shows footage from all 3 players:

https://github.com/ppy/osu/assets/37965983/87779016-157d-492a-9570-da987ddeca07

Obviously 3 is a small sample size, but I do believe this should be looked into and potentially changed. If necessary, I can try to gather more data points from first-time install users, but hopefully the video above represents the issue well.

**Reply by peppy** [reply]
It's funny how the users have all other games, AKA anything not discord set that low. Maybe the issue here is that discord is too quiet.

**Reply by DigitalHypno** [reply]
Forgot to mention: since we have a first install setup menu now, is it possible to include the volume sliders right when the player opens the game (along with the language select screen)?

As a side note, this issue is likely more prevalent here (despite being the same volume level as other games) because most games don't open with such an upbeat menu song.

**Reply by peppy** [reply]
I can only speak for myself but I have to basically set every game to 10-50% volume i'm when first starting to play.

This isn't osu! specific, and likely has a lot to do with discord voice's relative volume.

I'm not saying we shouldn't fix it, but I also think that going too low will negatively affect users that actually use system volume correctly.

I've discussed dropping the game volume to 50% by default, but there was some pushback from other developers IIRC.
